### ISSUE TITLE
DVD 576p releases treated being imported as Bluray-576p

### ISSUE THREAD

**Issue by edgarsuit** [Radarr/Radarr#7297]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Grab a DVDRip release with PAL resolution (or larger than standard NTSC resolution) that is clearly tagged as "DVDRip" in the release name, Radarr looks at the file resolution and assumes it's Bluray-576p rather than at the release name.

### Expected Behavior

If a release is tagged as a DVDRip, this should trump quality assumptions made based on file resolution unless there is a big discrepancy (i.e., DVDRip-tagged release with 1920x1080 file resolution)

### Environment

```markdown
- OS: Ubuntu 20.04
- Radarr: 4.1.0.6175
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Chrome
```

### What branch are you running?

Master

### Trace Logs?

...
2022-05-16 17:19:14.6|Debug|VideoFileInfoReader|Getting media info from /media/movies/Movies (Radarr)/84C MoPic (1989) (imdb-tt0096744)/84.MoPic.1989.DVDRip.x264-HANDJOB.mkv
2022-05-16 17:19:14.6|Trace|AggregateLanguage|Considering Languages Unknown (Filename) from FileName
2022-05-16 17:19:14.6|Trace|AggregateLanguage|Considering Languages English (MediaInfo) from MediaInfo
2022-05-16 17:19:14.6|Debug|AggregateLanguage|Selected languages: English
2022-05-16 17:19:14.6|Trace|AggregateQuality|Considering Source DVD (Tag) Resolution 480 (Fallback) Revision v1 from FileName
2022-05-16 17:19:14.6|Trace|AugmentQualityFromMediaInfo|Resolution 694x568 considered 576p
2022-05-16 17:19:14.6|Trace|AggregateQuality|Considering Source UNKNOWN (Default) Resolution 576 (MediaInfo) Revision  from MediaInfo
2022-05-16 17:19:14.6|Trace|AggregateQuality|Selected Source DVD (Tag) Resolution 576 (MediaInfo) Revision v1
2022-05-16 17:19:14.6|Warn|QualityFinder|Unable to find exact quality for DVD,  576, and NONE. Using Bluray-576p as fallback
2022-05-16 17:19:14.6|Debug|AggregateQuality|Using quality: Bluray-576p v1
...

[AB#3165](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3165)

**Comment by bakerboy448** [comment]
Note:  this is mostly due to the media info aggregator trumping file parsed information, which is valid.  

This can possibly be corrected with an order of operations fix in the logic to have dvd source taken into account with the low resolutions.

**Comment by austinwbest** [comment]
Im not so sure about that. Media info doesn't care about the source of DVD or Bluray so it isn't trumping DVD to Bluray because of media info telling it one or the other.

I believe as i mentioned yesterday the issue is in that radarr doesn't know DVD can be 576 in the quality finder. The quality defines will group DVD as a single resolution but since that resolution is 480 and the file is 576 it gets upped to Bluray576p

DVD needs to accept DVD480p and DVD576p i believe is what would resolve this.

Something like adding...
```public static Quality DVD => new Quality(32, "DVD", Source.DVD, 576);```

and then i think the find by source and resolution function would find it. Q or TA are more familiar with the c# than i am but they can confirm if my thinking is right and if it would be just adding that for a supported quality

**Comment by edgarsuit** [comment]
It did the same thing on a 576p WEB-DL file: The.Day.Reagan.Was.Shot.2001.NOW.WEB-DL.AAC2.0.H.264-QOQ.mkv

It assumed this was Bluray-576p.

**Comment by austinwbest** [comment]
i believe if the findbysourceandresolution function doesn't return a match, it fallsback to the closest resolution to it and this case bluray is the option for 576

Although web-dl might be a regex fixable one. 

Will wait for one of the C# gurus to chime in

**Comment by austinwbest** [comment]
@ta264 or @Qstick, is the suggestion a couple posts up the way to go about fixing this?

**Comment by andy-reeves** [comment]
Looking at the source it needs a new Quality definition for 576p DVDs (region 2 DVDs). Like:

`public static Quality DVD576p => new Quality(32, "DVD", Source.DVD, 576); //new`

DVD ought to have been DVD480p but probably cannot change this now.

**Comment by andy-reeves** [comment]
Hi,
I have a fix for this in a local branch 'fix-7297' off the develop branch. How do I publish my local branch to creat ethe pull request into develop please?

**Comment by andy-reeves** [comment]
Confirmed fixed in 4.5.1.7282
